source/developers/cook-books/how-tos/web-app-manifest-use-credentials.rst

```diff
@@ -0,0 +1,57 @@
+:is-up-to-date: True
+
+.. index:: Enable Use of Cookies and Credentials by the Web App Manifest, Web App Manifest, Manifest
+
+===================================================
+Enabling Use of Credentials by the Web App Manifest
```

Comment:
I'm thinking people's lookup terms when searching for what this article teaches might be more "web app manifest" than enabling using of credentials. We should front load those terms. 

If they know they need to enable credentials they don't even need this article.

I'd frame this whole article around to how to use a manifest with Crafter CMS and the title to be "Web manifest usage in Crafter CMS"